### PR TITLE
Apple page clone

### PR DESCRIPTION
**Pull request by eliasecasta**
In this project I built a clone webpage of the Apple webpage. I had to create the article required applying grids and background images, I also used flex box and grid.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        </div>
+        <div class="flex-items item2">Store</div>
+        <div class="flex-items item3">Mac</div>
+        <div class="flex-items item4">iPod</div>
+        <div class="flex-items item5">iPhone</div>
+        <div class="flex-items item6">iPad</div>
+        <div class="flex-items item7">iTunes</div>
+        <div class="flex-items item8">Support</div>
+        <div class="flex-items item9">
+          <span></span>
```
Comment (sumancrest0001) [inline index.html]: **STATUS: REQUIRED CHANGES
Hello @e71az,**
It looks almost perfect to me and most of the requirements are implemented but there are a couple of minor changes required. Let's fix them.
[Required] we have to add a link to each of the nav-items. [Link to the page](https://web.archive.org/web/20140301004610/http://www.apple.com/)

File: normalize.css [hunk 2]
```diff
@@ -0,0 +1,351 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
+
+/* Document
+   ========================================================================== */
+
+/**
+ * 1. Correct the line height in all browsers.
+ * 2. Prevent adjustments of font size after orientation changes in iOS.
+ */
+
```
Comment (sumancrest0001) [inline normalize.css]: [Required] Let's remove all the comments. If you want to reference them back then use git history.

**Review by sumancrest0001 — COMMENTED** [review]
[Required] the image-logo at the footer should be on the same line as other items. Currently, it is on another line. Have a look :point_down: 
Screenshot from 2020-05-20 11-27-28

It should be :point_down: 
Screenshot from 2020-05-20 11-27-41

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
[IMPORTANT] Stickler is not passing, I think there is some problem with it. Let's fix.

After making the requested changes, request for another review.
**if you need help or have queries regarding the review, DM me on slack Suman Shrestha**
**Happy coding !!!**

**Review by trekab — APPROVED** [review]
# Approved!
Hi @e71az, you have done an impressive job on this project! :clap:  You may merge to master.
_All the best on your next projects!_ :wink:
__Treasure Kabareebe__

**Pull request merged**